synapse/models/infotech.py

```diff
@@ -1517,6 +1524,9 @@ def getModelDefs(self):
                                            'uniq': True, 'sorted': True, 'split': ','}), {
                         'doc': 'An array of ATT&CK tactics that include this technique.',
                     }),
+                    ('data:components', ('array', {'type': 'it:mitre:attack:data:component', 'uniq': True}), {
```

Comment:
should this array be sorted as well?

Reply:
Since `it:mitre:attack:data:component` is a guid type now, I don't think there's much value in sorting these. (It sorts based on the primary property, right?)

Reply:
Yeah in this case its mostly just to prevent updating the index/stored value if the property is re-set to the same value in a different order.